### PR TITLE
VyOS Importer and Fetcher

### PR DESCRIPTION
**Pull request by mire-all-rashly**
Added a VyOS import tool and fetch script.  Initial level support for the basic/common features.

### CODE REVIEW

File: importers/nmdb-import-vyos/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+EXAMPLES
+======== 
+``` 
+import-tool --device-id server tool-data.txt 
+command | 
+import-tool --pipe --device-id server tool-data.txt 
+``` 
+
+See Also: `related-tool (8)`
```
Comment (marshall-sg) [inline importers/nmdb-import-vyos/README.md]: Needs to be specific to tool.

File: importers/nmdb-import-vyos/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#install_bin(${TOOL_NAME} ${PROJECT_NAME})
+#install_man(${TOOL_NAME} ${PROJECT_NAME})
+
+#install(
+#  FILES
+#    example.conf
+#  OPTIONAL
+#  COMPONENT ${PROJECT_NAME}
+#  DESTINATION ${NETMELD_CONF_DIR}
+#  )
```
Comment (marshall-sg) [inline importers/nmdb-import-vyos/CMakeLists.txt]: Add hooks to install tool and man pages.

File: importers/nmdb-import-vyos/Parser.cpp [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+    ;
+
+  login =
+    qi::lit("login") >> startBlock >
+    *(user [pnx::bind(&Parser::noteCredentials, this)]) > stopBlock
+    ;
+
+  user =
+    qi::lit("user") > token
+      [pnx::bind([&](std::string val){creds[1] = val;}, qi::_1)] >
```
Comment (mire-all-rashly) [inline importers/nmdb-import-vyos/Parser.cpp]: @cctechwiz FYI, you can use lambdas in the semantic action.

File: importers/nmdb-import-vyos/README.md (hunk 1 above)
Comment (mire-all-rashly) [reply importers/nmdb-import-vyos/README.md]: Fixed

File: importers/nmdb-import-vyos/CMakeLists.txt (hunk 2 above)
Comment (mire-all-rashly) [reply importers/nmdb-import-vyos/CMakeLists.txt]: Fixed.

**Pull request merged**